README.md

```diff
@@ -58,20 +22,14 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [MotoMate] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+- Motomate is a booking app that allows users to easily book and reserve motorcycles online. With Motomate, users can log in and access a wide range of motorcycles to choose from. You can check the back-end of this project [here.](https://github.com/NatanimA/MotoMate-Backend-Api)
```

Comment:
- [ ]  Please, kindly refactor these functions `MakeReservation` and `Login` in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const MakeReservation= () => {
  // ...
}
```

```javascript
const Login= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 2 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/223079933-8148399a-3eea-414f-89fb-08cf7e0d92c4.png)